zpls_echograms/README.md

```diff
@@ -1,104 +1,193 @@
+folder named `GA`, the folders would look like:
+
 ```
-./GI/ZPLSG_sn55067/DATA/201610
-        ./GI/ZPLSG_sn55067/DATA/201610/16101401.01A
-        ./GI/ZPLSG_sn55067/DATA/201610/16101402.01A
-        ./GI/ZPLSG_sn55067/DATA/201610/16101403.01A
-        ./GI/ZPLSG_sn55067/DATA/201610/*.01A
-        ./GI/ZPLSG_sn55067/DATA/201610/16101417.XML
-./GI/ZPLSG_sn55067/DATA/201611
-        ./GI/ZPLSG_sn55067/DATA/201611/16111401.01A
-        ./GI/ZPLSG_sn55067/DATA/201611/16111402.01A
-        ./GI/ZPLSG_sn55067/DATA/201611/16111403.01A
-        ./GI/ZPLSG_sn55067/DATA/201611/*.01A
+./GA/ZPLSG_sn55067/DATA/201610
+  ./GA/ZPLSG_sn55067/DATA/201610/16101401.01A
+  ./GA/ZPLSG_sn55067/DATA/201610/16101402.01A
+  ./GA/ZPLSG_sn55067/DATA/201610/16101403.01A
+  ./GA/ZPLSG_sn55067/DATA/201610/*.01A
+  ./GA/ZPLSG_sn55067/DATA/201610/16101417.XML
+./GA/ZPLSG_sn55067/DATA/201611
+  ./GA/ZPLSG_sn55067/DATA/201611/16111401.01A
+  ./GA/ZPLSG_sn55067/DATA/201611/16111402.01A
+  ./GA/ZPLSG_sn55067/DATA/201611/16111403.01A
+  ./GA/ZPLSG_sn55067/DATA/201611/*.01A
 ```
 
-**Example 1:**  
-```python3    zpls_echogram.py    ./GI/ZPLSG_sn55067/DATA    ./GI/ZPLSG_sn55067/DATA/201610/16101417.XML    20161115    20161116    ./processed_folder```
+For the EK60, the data is organized by OOI technicians in folders based on the year, month and day (e.g. `2017/01/01`).
+Files are named by the instrument according to the date and time the file was created (e.g. 
+`OOI-D20170831-T113333.raw`). If the ZPLS data from the [Coastal Endurance, Oregon Offshore Cabled Shallow Profiler 
+Mooring, 200 m Platform](https://rawdata.oceanobservatories.org/files/CE04OSPS/PC01B/ZPLSCB102_10.33.10.143) 
+from 2017 was downloaded to local folder named `CE04`, the folders would look like:
+
+```
+./CE04/PC01B/ZPLSCB102_10.33.10.143/2017/01/01/
+  ./CE04/PC01B/ZPLSCB102_10.33.10.143/2017/01/01/OOI-D20170101-T000000.bot
+  ./CE04/PC01B/ZPLSCB102_10.33.10.143/2017/01/01/OOI-D20170101-T000000.idx
+  ./CE04/PC01B/ZPLSCB102_10.33.10.143/2017/01/01/OOI-D20170101-T000000.raw
+./CE04/PC01B/ZPLSCB102_10.33.10.143/2017/01/02
+  ./CE04/PC01B/ZPLSCB102_10.33.10.143/2017/01/02/OOI-D20170102-T000000.bot
+  ./CE04/PC01B/ZPLSCB102_10.33.10.143/2017/01/02/OOI-D20170102-T000000.idx
+  ./CE04/PC01B/ZPLSCB102_10.33.10.143/2017/01/02/OOI-D20170102-T000000.raw
+```
 
-The code will process the data from 11/15/2016 to 11/16/2016. The .nc file and .png plot will be stored at ./processed_folder.
+For the AZFP example above, the root data directory would be `./GA/ZPLSG_sn55067/DATA`. For the EK60 example, the root 
+data directory would be `./CE04/PC01B/ZPLSCB102_10.33.10.143/2017`. 
+
+Processed data is created in a folder of the users choice. We recommend placing the data at the same level as the root 
+data directory in a separate folder called `processed`. The data in the `processed` folder is organized into subfolders 
```

Comment:
data in the `processed` folder **are** organized

Reply:
Changes made